endurance-page-cache.php

```diff
@@ -1082,35 +1103,49 @@ public function update_htaccess() {
 		 * @return string
 		 */
 		public function htaccess_contents_rewrites( $rules ) {
-			if ( false === is_numeric( $this->cache_level ) ) {
-				$this->cache_level = 2;
-			}
-
-			if ( $this->cache_level > 3 ) {
-				$this->cache_level = 3;
-			}
-
 			$base      = wp_parse_url( trailingslashit( get_option( 'home' ) ), PHP_URL_PATH );
 			$cache_url = $base . str_replace( get_option( 'home' ), '', WP_CONTENT_URL . '/endurance-page-cache' );
 			$cache_url = str_replace( '//', '/', $cache_url );
-			$additions = "\n
-				<ifModule mod_headers.c>\n" . '
-					Header set X-Endurance-Cache-Level "' . $this->cache_level . '"' . "\n" . '
-					Header set X-nginx-cache "WordPress"' . "\n
-				</ifModule>\n";
+
+			$additions = 'Options -Indexes' . PHP_EOL;
+
+			$additions .= <<<HTACCESS
+<IfModule mod_headers.c>
+	Header set X-Endurance-Cache-Level "{$this->cache_level}"
+	Header set X-nginx-cache "WordPress"
+</IfModule>
+HTACCESS;
+
+			$additions .= PHP_EOL;
 
 			if ( $this->use_file_cache() ) {
-				$additions .= 'Options -Indexes ' . "\n" . '
-				<IfModule mod_rewrite.c>
-					RewriteEngine On
-					RewriteBase ' . $base . '
-					RewriteRule ^' . $cache_url . '/ - [L]
-					RewriteCond %{REQUEST_METHOD} !POST
-					RewriteCond %{QUERY_STRING} !.*=.*
-					RewriteCond %{HTTP_COOKIE} !(wordpress_test_cookie|comment_author|wp\-postpass|wordpress_logged_in|wptouch_switch_toggle|wp_woocommerce_session_) [NC]
-					RewriteCond %{DOCUMENT_ROOT}' . $cache_url . '/$1/_index.html -f
-					RewriteRule ^(.*)$ ' . $cache_url . '/$1/_index.html [L]
-				</IfModule>' . "\n";
+				$additions .= <<<HTACCESS
```

Comment:
@wpscholar I don't think we ever nailed down the issue with `wp-env` but [we ended up switching](https://github.com/bluehost/bluehost-wordpress-plugin/pull/103/commits/badfeb8321ce910ee807c1db39b87d41b32afa03) to output buffers from heredoc because of fatal errors in `wp-env` that were tanking tests. We're not running `wp-env` here so this is probably fine, but when this rolls lets keep our ear to the ground for any issues as this goes to so many kinds of environments.